### PR TITLE
track calendar usage

### PR DESCRIPTION
**Pull request by jamesgorrie**
My general feeling is that people don't really use the calendar - it'd be good to see if they do, and if so, how.

### CODE REVIEW

File: kahuna/public/js/components/gu-date-range/gu-date-range.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <div class="gu-date-range" ng:init="ctrl.showOverlay = false">
-    <div class="gu-date-range__display" ng:click="ctrl.showOverlay = !ctrl.showOverlay">
+    <button class="gu-date-range__display"
```
Comment (theefer) [inline kahuna/public/js/components/gu-date-range/gu-date-range.html]: `button` = thanks :octopus:

**Comment by theefer** [comment]
:+1:

PS: I use it :-)

**Comment by paperboyo** [comment]
Anecdata: I rarely do. And whenever I try, I spend considerable time setting a simple range. Every time I have to learn again if I set the start on the left or on the right (why some months/years are unavailable in the dropdown or greyed out) and why do I have to care for it and it doesn't work both ways. There are some minor glitches here as well: https://github.com/guardian/grid/issues/672.

**Comment by akash1810** [comment]
@paperboyo would date filter search syntax be helpful at all?

**Comment by paperboyo** [comment]
@akash1810 Maybe it should be there one day for consistency and ease of use (chips?). But I'm okay using a picker randomly, just so I can quickly modify a URL path for now. Admittedly, I'm not a huge date ranges user. Thx!

**Pull request merged**